src/main/java/step3/RacingGame.java

```diff
@@ -0,0 +1,36 @@
+package step3;
+
+import java.util.Random;
+import java.util.List;
+
+public class RacingGame {
+    //  게임 진행하는 클래스 https://github.com/next-step/java-racingcar/pull/909/commits/db637e88d70bc34ed8f1c854c55e8ce82eb3a19c
+    private List<RacingCar> RacingCars;
```

Comment:
변수명은 lowerCamelCase 로 선언해주세요 ! 
https://google.github.io/styleguide/javaguide.html#s5-naming
```suggestion
    private List<RacingCar> racingCars = new ArrayList<>();
```